src/NzbDrone.Core/Download/Clients/DownloadStation/Proxies/DiskStationProxyBase.cs

```diff
@@ -172,40 +112,62 @@ private string GenerateSessionCacheKey(DownloadStationSettings settings)
 
         private string AuthenticateClient(DownloadStationSettings settings)
         {
-            var arguments = new Dictionary<string, object>
-            {
-                 { "api", "SYNO.API.Auth" },
-                 { "version", "1" },
-                 { "method", "login" },
-                 { "account", settings.Username },
-                 { "passwd", settings.Password },
-                 { "format", "sid" },
-                 { "session", "DownloadStation" },
-             };
-
             var authInfo = GetApiInfo(DiskStationApi.Auth, settings);
 
-            var requestBuilder = BuildRequest(settings, authInfo, HttpMethod.GET);
+            var requestBuilder = BuildRequest(settings, authInfo, "login", 2);
```

Comment:
'Was working'? I'm asking because on master&develop you send v=1, see: https://github.com/Sonarr/Sonarr/blob/cf306f4aba051c0a8011dc03b047993297cef8e6/src/NzbDrone.Core/Download/Clients/DownloadStation/Proxies/DiskStationProxyBase.cs#L98